sycl/test-e2e/USM/align.cpp

```diff
@@ -0,0 +1,102 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+// UNSUPPORTED: gpu
+
+// E2E tests for annotated USM allocation functions with alignment arguments
```

Comment:
Unified Runtime [documentation says](https://oneapi-src.github.io/unified-runtime/core/api.html#urusmdevicealloc) that there should be an error in that case. I think that we should submit a bug report to the corresponding GitHub repo if we don't see that error.

Reply:
Respective issue opened here: https://github.com/intel/llvm/issues/12638